.circleci/config.yml

```diff
@@ -938,6 +949,7 @@ workflows:
             - requestsgevent
             - sqlalchemy
             - sqlite3
+            - test_build_alpine
```

Comment:
👍 thank you

Reply:
I'd consider adding all versions we support on alpine, but this is fine.

Reply:
Indeed, the compatibility issue is with their libc, not with any Python version specifically, so I don't think it's worth it.